tests/test_default_headers.py

```diff
@@ -12,79 +12,69 @@ async def app(scope, receive, send):
     await send({"type": "http.response.body", "body": b"", "more_body": False})
 
 
-def test_default_default_headers():
```

Comment:
I think the readability and maintenance of these tests would be better off if we keep separate and easy-to-understand test functions for each case. There's some duplication obviously, but it's mostly harmless.

Reply:
I kept them seperated, just added a `# fmt: off` just to have the Config on one line, can remove it of course I just felt it was more readable this way despite the length